validation/src/test/scala/hmda/validation/rules/lar/macro/Q047Spec.scala

```diff
@@ -0,0 +1,38 @@
+package hmda.validation.rules.lar.`macro`
+import com.typesafe.config.ConfigFactory
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.rules.AggregateEditCheck
+import hmda.validation.rules.lar.`macro`.MacroEditTypes.LoanApplicationRegisterSource
+
+class Q047Spec extends MacroSpec {
+
+  val config = ConfigFactory.load()
+  val multiplier = config.getDouble("hmda.validation.macro.Q047.numOfLarsMultiplier")
+
+  val testLars = lar100ListGen.sample.getOrElse(Nil)
+  val sampleSize = testLars.size
+  def relevantLar(lar: LoanApplicationRegister) = lar.copy(actionTakenType = 2)
+  def irrelevantLar(lar: LoanApplicationRegister) = {
+    lar.copy(actionTakenType = 4).copy(preapprovals = 1)
+  }
+
+  property(s"be valid if single family and withdrawn < $multiplier * total") {
```

Comment:
I think the description here should be "preapproval requested and application withdrawn" (unless "single family" is specified somewhere? I may have missed it)